content/configuration/dev-server.md

```diff
@@ -312,7 +439,36 @@ overlay: {
 }
 ```
 
-## `devServer.port` - CLI only
+## `devServer.pfx`
+
+`string`
+
+Path to a SSL pfx file.
+
+```js
+pfx: '/path/to/file.pfx'
+```
+
+Usage via the CLI
+```
+webpack-dev-server --pfx /path/to/file.pfx
+```
+
+## `devServer.pfxPassphrase`
+
+`string`
+
+The passphrase to a SSL PFX file.
+```js
+pfxPassphrase: 'passphrase'
+```
+
+Usage via the CLI
+```
+webpack-dev-server --pfx-passphrase passphrase
+```
+
```

Comment:
It should be two line breaks between each section -- it seems on some you did add two but at least a few of these only have one.